### PR TITLE
Pr misc updates to prov

### PR DESCRIPTION
**Pull request by mattsm**

### CODE REVIEW

File: tests/rootfs_boot.py [hunk 1]
```diff
@@ -183,11 +184,18 @@ def boot(self, reflash=True):
 
             ips = []
             for not_used in range(5):
-                ips = [board.get_interface_ipaddr(board.wan_iface)]
+                ip = board.get_interface_ipaddr(board.wan_iface)
+                assert ipaddress.IPv4Address(ip.decode('utf-8')) in prov.cm_network
```
Comment (mgualco-contractor) [inline tests/rootfs_boot.py]: Linked to the error reporting improvements, could you add an err message string to the asserts?

Comment (mattsm) [reply tests/rootfs_boot.py]: I'm not seeing this link...

File: devices/debian_isc.py [hunk 2]
```diff
@@ -115,6 +113,8 @@ class "HOST" {
   }
   pool {
     range ###MTA_START_RANGE### ###MTA_END_RANGE###;
+    option docsis-mta.dhcp-server-1 ###MTA_DHCP_SERVER1###;
```
[the remark this answers is not in the document]
Comment (mattsm) [reply devices/debian_isc.py]: It's in a follow up commit. I tried to break them up logically.

File: tests/rootfs_boot.py (hunk 1 above)
Comment (mgualco-contractor) [reply tests/rootfs_boot.py]: linked i.e. with reference to improving error reporting

Comment (mgualco-contractor) [reply tests/rootfs_boot.py]: yep, assert, "this has happened maybe do this to fix it"

**Pull request merged**